Examples/Swift/PointHeatmapExample.swift

```diff
@@ -0,0 +1,54 @@
+
+import Mapbox
+
+@objc(PointHeatmapExample_Swift)
+
+class PointHeatmapExample_Swift: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.darkStyleURL(withVersion: 9))
+        mapView.delegate = self
+        
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Parse GeoJSON data from USGS on earthquakes in the past week.
+        DispatchQueue.global().async {
+            guard let url = URL(string: "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_week.geojson") else { return }
+            DispatchQueue.main.async {
+                self.displayEarthquakes(url: url, style: style)
+            }
```

Comment:
Can you say more about what the intention is here, re: threading? It appears that you’re creating the URL on a background thread, then doing the actual fetching and rendering on the main thread.